app/models/tutor.rb

```diff
@@ -1,8 +1,6 @@
 class Tutor < ApplicationRecord
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
 
-  before_save { email.downcase! }
```

Comment:
Just wondering if there was another reason. Since emails are understood by mailers if they have uppercase no differently I think its fine to leave.